vxdetector/files_manager.py

```diff
@@ -6,34 +6,46 @@
 
 
 def tmp_dir(path, temp_path):
-    """erstellt tmp verzeichnis; sollte es auch deleten"""
+    # erstellt tmp verzeichnis, fals es schon existiert
+    # wird es geloescht
     if os.path.exists(temp_path):
         shutil.rmtree(temp_path)
     else:
-        temp_path = tempfile.mkdtemp(suffix=None, prefix='tmp_files_',
-                                     dir=path) + '/'
+        temp_path = f"{tempfile.mkdtemp(prefix='tmp_files_', dir=path)}/"
+
     return temp_path
 
 
 def get_lib():
-    """findet die richtigen ordner/dateien, damit das programm funktioniert"""
-    programm_path = os.path.dirname(__file__) + '/'
-    if os.path.exists(programm_path+'Output/'):
+    # findet die richtigen ordner/dateien, damit das programm funktioniert
```

Comment:
could you please translate to English, should we ever pass this code base to the UCSD guys